.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish image to DockerHub
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
⚠️ the `v0.24.0rc0` tag will be tagged as latest in this case
If you don't release any beta it's ok

Reply:
Following the history of this repo, we didn't have a beta version yet. I think it is ok to keep the current change. But you're right, if we have plans to release betas here we will have problems :)